REST/src/main/java/pl/jakubowskiprzemyslaw/REST/models/BoardsViewHandler.java

```diff
@@ -0,0 +1,24 @@
+package pl.jakubowskiprzemyslaw.REST.models;
+
+import org.springframework.stereotype.Component;
+import pl.jakubowskiprzemyslaw.tajgertim.models.view.BoardsView;
+
+import java.util.Optional;
+
+@Component
```

Comment:
I'll ask why you need this to be a component during demo. :-)

Reply:
:+1: